src/RA_Dlg_AchEditor.cpp

```diff
@@ -83,9 +83,9 @@ Dlg_AchievementEditor::Dlg_AchievementEditor() noexcept
     for (size_t i = 0; i < MAX_CONDITIONS; ++i)
     {
         if (i == 0)
-            _stprintf_s(m_lbxGroupNames[i], MEM_STRING_TEXT_LEN, _T("Core"));
+            Ensures(_stprintf_s(gsl::at(m_lbxGroupNames, i), MEM_STRING_TEXT_LEN, _T("%s"), _T("Core")) >= 0);
```

Comment:
what's the difference between
```
m_lbxGroupNames.at(i)
```
and
```
gsl::at(m_lbxGroupNames, i)
```
?

Reply:
No difference, it's just gsl::at can bounds check raw arrays. I can make it std::array